test-files/cases/hoot-rnd/network/conflicts/highway-023/Expected.osm

```diff
@@ -112,15 +107,8 @@
         <nd ref="-9"/>
         <nd ref="-10"/>
         <nd ref="-13"/>
-        <tag k="highway" v="road"/>
-        <tag k="hoot:status" v="1"/>
-        <tag k="hoot:id" v="-16"/>
-    </way>
-    <way visible="true" id="-15" timestamp="1970-01-01T00:00:00Z" version="1">
-        <nd ref="-1"/>
-        <nd ref="-2"/>
-        <nd ref="-3"/>
-        <nd ref="-35"/>
+        <nd ref="-11"/>
+        <nd ref="-12"/>
```

Comment:
Some tests contained the same number of ways but some of the nodes shifted from one way to another to put them more in line with the original input data.